tutorials/sexual_behaviour.md

```diff
@@ -0,0 +1,196 @@
+        - Sex workers over 30 have their number of sexual partners capped at 30. 
+- `update_sex_groups` changes sexually active peoples sexual behaviour groups based on the probabilities described above. 
+- `update_sex_behaviour_class` only needs to change the sex behaviour class of women, as there is only one category for men. For women, it may change if their sex worker status changes, or they move over an age threshold in this timestep.  
+
+### Risk Factors 
+
+There are a number of risk factors calculated for individuals and for the population. These risk factors may raise or lower risk behaviour, based on whether they are greater or less than 1. An individuals total personal risk is a product of risk factors. Most risk factors have both an `init_risk...` and `update_risk...` function to give initial and then subsequent values. The list of risk factors, and how to set them, is as follows:
+
+- `RISK_PERSONAL`: An individual risk factor that is given to each person. The value `population_risk_personal` in the data gives possible probabilities of a person being designated low risk, and is sampled during the `__init__`. For each person, `RISK_PERSONAL` is set to $10^{-5}$ with this probability, and $1$ otherwise. This is not updated during the simulation. 
+- `RISK_AGE`: Age related risk factor. Set this in `age_based_risk_options` in the data file; these options are sampled during `__init__` to give the age based risk factors for a given simulation.  
+- `RISK_ADC`: Risk factor associated with reduction of risk behaviour for a person with an AIDS Defining Condition. If a person is HIV positive and has an ADC then this is 0.2; otherwise it is 1. 
+- `RISK_BALANCE`: Behavioural changes imposed to balance out the number of male and female short term condomless sex partners if these become uneven. Calculates the number of short term partners of all men $S_m$ and of all women $S_w$. For a population of size $N$, we then calculate the normalised discrepancy $D = \frac{S_m + S_w}{N}$. 
+    - The balance factor is set for different thresholds of $D$:
+        - $D > 0.1 \implies R_b = 0.1$
+        - $D > 0.03 \implies R_b = 0.7$
+        - $D > 0.005 \implies R_b = 0.7$
+        - $D > 0.004 \implies R_b = 0.75$
+        - $D > 0.003 \implies R_b = 0.8$
+        - $D > 0.002 \implies R_b = 0.9$
+        - $D > 0.001 \implies R_b = 0.97$
+        - otherwise $R_b = 1$
+    - `RISK_BALANCE` is set to $R_b$ for the sex with the excess, and $R_b^{-1}$ for the other sex. 
+- `RISK_DIAGNOSIS`: Change in risk behaviour associated with recent HIV diagnosis. Under `risk_diagnosis` in the data file you can set the values it can take, and the probabilities of that (which are samples at the `__init__` for the simulation), as well as the `Period` which is the length of time after an HIV diagnosis for which this effect lasts. For anyone without a recent HIV diagnosis, this is 1. 
```

Comment:
```suggestion
- `RISK_DIAGNOSIS`: Change in risk behaviour associated with recent HIV diagnosis. Under `risk_diagnosis` in the data file you can set the values it can take and their probabilities (which are sampled at the `__init__` for the simulation), as well as the `Period` which is the length of time after an HIV diagnosis for which this effect lasts. For anyone without a recent HIV diagnosis, this is 1. 
```